style.css

```diff
@@ -3282,4 +3282,4 @@ article.panel-placeholder {
 	.twentyseventeen-front-page .entry-content blockquote.alignright {
 		margin-right: -20%;
 	}
-}
\ No newline at end of file
+}
```

Comment:
i did not even touch the code i just make 0.76em to 0.40em and thats it😒 how this is possible😕